src/content/posts/2022/bootstrap-5-2-1.md

```diff
@@ -0,0 +1,43 @@
+---
+author: mdo
+date: "2022-09-02T16:00:00Z"
+title: Bootstrap 5.2.1
+video:
+keywords:
+  - bootstrap
+  - release
+---
+
+Bootstrap v5.2.1 is here with fixes from our latest minor release, v5.2. As a patch release, these changes are limited to bug fixes, documentation updates, and some dependency updates.
+
+## Highlights
+
+- Added a `transparent` default hover border color CSS variable for buttons to fix a visual regression
+- Horizontal list groups with only one child now render the correct `border-radius`
+- Input groups have updated `z-index` values to ensure proper rendering of validated form fields
+- Modified the `list-group-item` selectors to better support nested imports of Bootstrap's CSS
+- Floating labels now reset their `text-align` to ensure consistent styling
+- Updated Autoprefixer to fix warnings of the `color-adjust` property (thanks to @julien-deramond on our team for reporting an issue upstream here)
+- Fixed incorrect `border-radius` values inside pagination components
+- Moved our search functionality out of the hidden sidebar on narrow viewports and mobile devices
+- Removed links to and mentions of Slack from across the codebase, as we intend to shutter Slack in favor of GitHub Discussions
+- Scrollspy threshold option is now configurable
+- Fixed modal event listeners during dismiss click, allowing you to once again click scrollbars without dismissing the modal
+- Reverted some tooltip plugin updates to prevent issues with `selector`, dynamic content, and disposed tooltips using `title`
+- Reintroduced proper `outline` styles for docs code samples and buttons when `:not(:focus-visible)`
+- Searching our docs on mobile has been improved with the search field always visible on narrow viewports
+- Removed links to Slack from our docs as we've closed it down in favor of GitHub Discussions
+
```

Comment:
Maybe something's missing based on https://github.com/twbs/bootstrap/commit/7688c84b8a881f1fe131b204d16ba13246d25da5 and https://github.com/twbs/bootstrap/commit/9e57dfadac9cd924c35b7ff5b3d82981b59502de.

Proposal:
`- Added a new 'Features with title' section in our [Features example](https://getbootstrap.com/docs/5.2/examples/features/) and fixed heading levels

Reply:
I skipped this one because it's not the big of an update, so for now I'll leave it out.